demo/time-picker-basic-demos.html

```diff
@@ -19,9 +19,30 @@ <h3>Value Resolution</h3>
       The <code>step</code> property specified in seconds affects the resolution of the time picker and visibility of the dropdown.
       If step is less than 15 mins dropdown is hidden
     </p>
-    <vaadin-demo-snippet id="time-picker-basic-demos-step-example">
+    <vaadin-demo-snippet id="time-picker-basic-demos-step-example" when-defined="vaadin-time-picker">
       <template preserve-content>
-        <vaadin-time-picker step="0.5" label="Duration (hh:mm:ss.fff)"></vaadin-time-picker>
+        <vaadin-combo-box id="combobox-step-selection" value="0.01" label="Step in minutes"></vaadin-combo-box>
+        <vaadin-time-picker id="time-picker-basic-demos-value-resolution" step="0.6" label="Duration (hh:mm:ss.fff)"></vaadin-time-picker>
+        <script>
+          window.addDemoReadyListener('#time-picker-basic-demos-step-example', function(document) {
+            const comboBox = document.querySelector('#combobox-step-selection');
+            const timePicker = document.querySelector('#time-picker-basic-demos-value-resolution');
+            comboBox.items = ['0.01', '0.5', '1', '15', '60', '120'];
```

Comment:
This is a good idea but imo these options are confusing since 0.01 minutes = 0.6 seconds and 0.5 minutes = 30 seconds which are a bit random.

We should use clearly understandable options here. Probably should use the separate values and labels to use exact second values with human readable visible options.

I suggest using values like this:
```suggestion
            comboBox.items = [
              {value: '0.5', label: '0.5 seconds (hh:mm:ss.fff)'},
              {value: '1', label: '1 second (hh:mm:ss)'},
              {value: '60', label: '1 minute (hh:mm)'},
              {value: String(60*15), label: '15 minutes (hh:mm)'},
              {value: String(60*30), label: '30 minutes (hh:mm)'},
              {value: String(60*60), label: '1 hour (hh:mm)'},
              {value: String(60*60*2), label: '2 hours (hh:mm)'}
            ];
```
And updating the combo box label to just `label="Step"` and setting `style="width: 250px"` on it so all the options are displayed nicely.

Reply:
Done

Reply:
Thanks. It looks better now but I found some issues. See my latest review.